flutter/lib/utils.dart

```diff
@@ -0,0 +1,33 @@
+//Flutter Packages
+import 'package:flutter/material.dart';
+//Third party packages
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:top_snackbar_flutter/custom_snack_bar.dart';
+import 'package:top_snackbar_flutter/top_snack_bar.dart';
+
+void showSuccess(BuildContext context, String message) {
+  showTopSnackBar(context, CustomSnackBar.success(message: message),displayDuration: Duration(seconds: 3));
+}
+
+void showError(BuildContext context, String message) {
+  showTopSnackBar(context, CustomSnackBar.error(message: message),displayDuration: Duration(seconds: 3));
+}
+
+void showInfo(BuildContext context, String message) {
+  showTopSnackBar(context, CustomSnackBar.info(message: message),displayDuration: Duration(seconds: 3));
+}
+
+void showLoader(){
+  EasyLoading.show(
+    //status: 'Processing...',
+    maskType: EasyLoadingMaskType.black,
+  );
+}
+
+void hideLoader(){
+  EasyLoading.dismiss();
+}
+
+bool isLoaderShowing(){
+  return EasyLoading.isShow;
+}
```

Comment:
**suggestion**: Rather than having these functions called from your provider you could communicate these as states and handle it in your views. As it is you are coupling UI with your provider (view model).